modules/material-management/src/pem_helpers.ts

```diff
@@ -92,7 +92,7 @@ export function chunk64(buff: Buffer) {
   const chunks: string[] = new Array(numChunks)
 
   for (let i = 0, o = 0; i < numChunks; ++i, o += chunkSize) {
-    chunks[i] = str.substr(o, chunkSize)
```

Comment:
`substr` is start, length and `slice` is start end :)
so `"length vs position".substr(5,3) == "h v"`
but `"length vs position".slice(5,3) == ""`

Reply:
You are absolutely right. I must have read "o" as "0" aka zero. I updated the PR